be/src/runtime/load_channel.h

```diff
@@ -71,6 +71,14 @@ namespace lake {
 class TabletManager;
 }
 
+struct LoadChannelOpenRequest {
```

Comment:
this is also the same as the RPC's controller, maybe name it `LoadChannelOpenController` or `LoadChannelOpenContext`

Reply:
fixed. rename it to `LoadChannelOpenContext`